.github/actions/build-package/action.yml

```diff
@@ -44,6 +44,7 @@ runs:
     - uses: cloudscape-design/actions/.github/actions/patch-local-dependencies@main
       with:
         path: ${{ github.workspace }}/${{ inputs.package }}
+        tarball-dir: ${{ github.workspace }}
```

Comment:
Should we include the branch name here? We can then default to main but allow a custom branch name as a parameter. In that case it should be possible to make a dry-run against a branch that is not yet merged.

Reply:
I'm not sure I understand the suggestion. This action (patch-local-dependencies) does not run a checkout and. Maybe this is a discussion for the next [PR](https://github.com/cloudscape-design/actions/pull/53) ?